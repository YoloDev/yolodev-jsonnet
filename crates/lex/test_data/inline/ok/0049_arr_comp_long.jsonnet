[x * y for x in [1, 2, 3] if true for y in [7, 8, 9]]
