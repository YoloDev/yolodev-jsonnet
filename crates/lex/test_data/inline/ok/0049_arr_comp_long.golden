SymbolLeftBracket@0:1 "["
Ident@1:2 "x"
Whitespace@2:3 " "
OpMul@3:4 "*"
Whitespace@4:5 " "
Ident@5:6 "y"
Whitespace@6:7 " "
KeywordFor@7:10 "for"
Whitespace@10:11 " "
Ident@11:12 "x"
Whitespace@12:13 " "
KeywordIn@13:15 "in"
Whitespace@15:16 " "
SymbolLeftBracket@16:17 "["
Number@17:18 "1"
SymbolComma@18:19 ","
Whitespace@19:20 " "
Number@20:21 "2"
SymbolComma@21:22 ","
Whitespace@22:23 " "
Number@23:24 "3"
SymbolRightBracket@24:25 "]"
Whitespace@25:26 " "
KeywordIf@26:28 "if"
Whitespace@28:29 " "
KeywordTrue@29:33 "true"
Whitespace@33:34 " "
KeywordFor@34:37 "for"
Whitespace@37:38 " "
Ident@38:39 "y"
Whitespace@39:40 " "
KeywordIn@40:42 "in"
Whitespace@42:43 " "
SymbolLeftBracket@43:44 "["
Number@44:45 "7"
SymbolComma@45:46 ","
Whitespace@46:47 " "
Number@47:48 "8"
SymbolComma@48:49 ","
Whitespace@49:50 " "
Number@50:51 "9"
SymbolRightBracket@51:52 "]"
SymbolRightBracket@52:53 "]"
Whitespace@53:54 "\n"
